cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -141,17 +141,17 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+start_addon /etc/kubernetes/addons/namespace.yaml 100 10 "" &
```

Comment:
Given that this was missed twice, a comment stating what it's doing would be useful. Same goes for the loop below. 

This is creating the namespace that the addons will go into.